setup.py

```diff
@@ -55,7 +55,7 @@
 if py_version < '2.7':
     REQUIRED_PACKAGES.append('argparse>=1.2.1')
 
-_APITOOLS_VERSION = '0.4.15'
+_APITOOLS_VERSION = '0.5.0'
```

Comment:
can we set this to either 0.4.99 or 0.5.0a1 for now? i still want to see what happens with gsutil testing against the new version (in particular with the new oauth2client).